client/my-sites/posts/post-list.jsx

```diff
@@ -277,7 +277,7 @@ var Posts = React.createClass( {
 				/>
 			);
 		} else {
-			if ( this.props.loading || ! this.props.sites.fetched ) {
+			if ( this.props.loading || ! this.props.haveSites ) {
```

Comment:
Looking at the surrounding code might help. We want to determine whether to show a list of posts, a list of placeholders, or a "no content" message. Posts aside, how do we want this to depend on `sites`?